### PR TITLE
Migrated ConfigurationManagementServiceImplTest to use new annotations issue#1165 

### PR DESCRIPTION
**Pull request by Gcolon021**
@carlspring I believe this is completed. 

Edit: Still WIP

### CODE REVIEW

File: strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImplTest.java [hunk 1]
```diff
@@ -140,9 +144,10 @@ public void testGetGroupRepositories()
 
     @Test
     @ExtendWith(RepositoryManagementTestExecutionListener.class)
-    public void testGetGroupRepositoriesContainingRepository(@NullRepository(repositoryId = REPOSITORY_RELEASES_1) Repository releases1,
-                                                             @TestRepository.Group(repositories = REPOSITORY_RELEASES_1)
-                                                             @NullRepository(repositoryId = REPOSITORY_GROUP_1) Repository releasesGroup)
+    public void testGetGroupRepositoriesContainingRepository(
+            @NullRepository(repositoryId = REPOSITORY_RELEASES_1) Repository releases1,
+            @TestRepository.Group(repositories = REPOSITORY_RELEASES_1)
+            @NullRepository(repositoryId = REPOSITORY_GROUP_1) Repository releasesGroup)
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImplTest.java]: Could you please preserve the original formatting? (Here and below). Thanks!

Comment (Gcolon021) [reply strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImplTest.java]: Copy that!

**Comment by ptirador** [comment]
@carlspring: I think this should be closed, because it has already been migrated,

**Comment by carlspring** [comment]
@ptirador : Thanks for following up on this and for letting me know!

**Pull request closed without merging**